ts/.eslintrc

```diff
@@ -37,6 +37,7 @@
         "no-undefined": 0,
         "no-underscore-dangle": 0,
         "no-unused-expressions": 0,
+        "no-use-before-define": 0,
```

Comment:
@bre1470 Do we need to disable this? I am rather in favor of keeping this rule and instead move the `isNumber` to before it is used.